visual-tests/src/test-fixtures/series/stacked-bar.js

```diff
@@ -117,23 +116,23 @@
               return d.datum.x;
           })
           .yLabel(function(d) { return d.datum.yProperty + ' : ' + d.datum.yValue; })
-          .snap(pixelSnap);
+          .snap(pixelSnap.bind(null, stackedData));        
 
         // Add it to the chart
         var multi = fc.series.multi()
             .xScale(x)
             .yScale(y)
-            .series([stack, crosshair])
+            .series([stackedBar, crosshair])
             .mapping(function(series) {
                 switch (series) {
-                    case stack:
+                    case stackedBar:
                         return this;
                     case crosshair:
-                        return this.crosshair;
+                        return [];
```

Comment:
This pattern will break if a full re-render occurs. Better to use a captured variable or augment the data with an additional property.